src/loaders/common/loader_common_names.py

```diff
@@ -83,4 +84,21 @@
 MASH_METADATA = 'mash_run_metadata.json'
 
 # The fatal error file created if a data file cannot be successfully processed
-FATAL_ERROR_FILE = "fatal_error.json"
\ No newline at end of file
+FATAL_ERROR_FILE = "fatal_error.json"
+
+# The summary files for gtdb_tk tool
+GTDBTK_AR53_SUMMARY_FILE = "gtdbtk.ar53.summary.tsv"
+GTDBTK_BAC120_SUMMARY_FILE = "gtdbtk.bac120.summary.tsv"
```

Comment:
@Tianhao-Gu do these file names change from release to release? I'm wondering if the tools may break on older or future releases

Reply:
I never tried other gtdb-tk versions. But in GTDB release 202, seems like they are named as ar122 and bac120. I guess if we want to be completely safe, we can get find files ending with `summary.tsv`

Reply:
Or even more safe `gtdbtk.*.summary.tsv`. Given the tool output dirs are isolated that seems reasonable.

Reply:
https://github.com/kbase/collections/issues/330

Reply:
Can you 
* delete the old hardcoded file name constants
* make a shared function for finding the gtdbk output files rather than duplicating the code

Reply:
Updated!

Reply:
:+1: